src/operations/validate_collection.ts

```diff
@@ -18,8 +18,9 @@ export class ValidateCollectionOperation extends CommandOperation<Document> {
   collectionName: string;
   command: Document;
 
-  constructor(admin: Admin, collectionName: string, options: ValidateCollectionOptions) {
```

Comment:
Removing the overloads in admin.ts validateCollection resulted in this change b/c the argument fixing logic is gone, we can do this in the helper or here, let me know which is best